tsconfig.json

```diff
@@ -1,17 +1,18 @@
 {
     "compilerOptions": {
         "declaration": true,
+        "allowSyntheticDefaultImports": true,
         "esModuleInterop": true,
         "jsx": "react",
         "lib": ["ESNext", "DOM"],
-        "module": "CommonJS",
+        "module": "ESNext",
```

Comment:
This should be a specific ratified specification version, [remark uses ES2020](https://github.com/remarkjs/remark/blob/bc55caa4cc9fc6ae317c3e79a6f7269035cdc3c8/tsconfig.json) so seems sensible to match that

Reply:
Good point, thank you!